app/views/shared/fields/_note.html.erb

```diff
@@ -46,7 +46,7 @@
         elsif note_type != nil && !note_type.blank?
           notes[note_type.titleize] = note
         else
-          notes['General Note'] = note
+          notes['General Note'.to_s] = note
```

Comment:
I'm not following turning a String into a String again as the solution. Apologies if I'm missing something.

I believe you noted the problem is that if there are more than one instance of 'General Note' then all but one are being overwritten by this assignment.

Rather than use '=' assignment, is this an instance where using '<<' assignment would be correct? This way you would never write over a previously stored value, you're just appending values. In fact, shouldn't we be doing this is all 3 conditional scenarios: https://github.com/ucsdlib/damspas/blob/feature/note_missing/app/views/shared/fields/_note.html.erb#L44-L50